### ISSUE TITLE
'Laste updated at' is showing time in the future

### ISSUE THREAD

**Issue by JokerQyou** [gogits/gogs#1687]
![image](https://cloud.githubusercontent.com/assets/1465267/10096279/c696838c-63a2-11e5-8785-f4918f76b2ee.png)

As you can see in the screenshot, I've just imported a mirrored repository, but the `last updated at` is showing time in the future. I'm in GMT+8 and the time this repo was imported is `2015-09-25 16:28` local time.

**Comment by JokerQyou** [comment]
Here is the result of `date` command on my server.

```bash
$ date
Fri Sep 25 16:35:41 CST 2015
```

**Comment by Unknwon** [comment]
Thanks your feedback!

1. What is your Gogs version?
2. Which database are you using?

**Comment by JokerQyou** [comment]
I'm using the binary build downloaded from official site with MySQL database.

**Comment by JokerQyou** [comment]
Version string from page footer is `0.6.9.0903 Beta`

**Comment by Unknwon** [comment]
Thanks your info!

Can you upgrade to `master` so we are on the same page?

**Comment by BurakDev** [comment]
Same issue on 0.6.9.0903 using SQLite3 storage

**Comment by JokerQyou** [comment]
Ok, I've not upgraded to `master` yet, but I think this seems to be a one-time issue after the first installation.

At first I transferred a Gogs server from Linode to Aliyun, and forgot to migrate the MySQL database. And I reinstalled this new server with Gogs running behind nginx (under `/projects/` path), and the styles, js and other resources were broken when I launched the installation page, thus leading me to use SQLite database (because without styling I can't really tell which was selected).
Then, right before I first reported this issue, I found I was using SQLite instead of MySQL, and I ran another re-installation. This issue then was found right after the installation finished as I mirrored my first repository.

Now I don't see any more time issue, I've mirrored some more repositories just now and they're ok.
